src/views/teachers/landing/landing.jsx

```diff
@@ -82,19 +82,25 @@ var Landing = injectIntl(React.createClass({
                         <h3 id="guides-header"><FormattedMessage id="teacherlanding.guidesTitle" /></h3>
                         <FlexRow className="guides-and-tutorials">
                             <div>
-                                <img src="/svgs/teachers/resources.svg" alt="resources icon" />
+                                <a href="https://scratch.mit.edu/help/">
+                                    <img src="/svgs/teachers/resources.svg" alt="resources icon" />
+                                </a>
                                 <p>
                                     <FormattedHTMLMessage id="teacherlanding.helpPage" />
                                 </p>
                             </div>
                             <div>
-                                <img src="/svgs/teachers/tips-window.svg" alt="tips window icon" />
+                                <a href="https://scratch.mit.edu/projects/editor/?tip_bar=home">
```

Comment:
same with this one https://github.com/LLK/scratch-www/blob/develop/src/views/teachers/landing/l10n.json#L18